src/GosuWrapper.cpp

```diff
@@ -0,0 +1,233 @@
+#include <Gosu/Gosu.hpp>
+
+
+extern "C" {
+  #include <Gosu/Gosu.h>
+
+  void Gosu_gl_z(double z, void function(void *data), void *data)
+  {
+    std::function<void (void)> callback;
+    callback = [=](void) { function(data); };
+
+    Gosu::Graphics::gl(z, callback);
+  }
+
+  void Gosu_gl(void function(void *data), void *data)
+  {
+    std::function<void (void)> callback;
+    callback = [=](void) { function(data); };
+
+    Gosu::Graphics::gl(callback);
+  }
+
+  Gosu_Image *Gosu_render(int width, int height, void function(void *data), void *data, unsigned image_flags)
+  {
+    std::function<void (void)> callback;
+    callback = [=](void) { function(data); };
```

Comment:
Using `(void)` to denote an empty parameter list instead of `()` is a C-ism and deprecated in C++. Even though this is an `extern "C"` area, this code is compiled as C++, and doesn't need the voids.

Reply:
:book: Ah, interesting.
:heavy_check_mark: should be corrected now.